templates/node.yaml

```diff
@@ -100,6 +100,8 @@ spec:
             - --endpoint=$(CSI_ENDPOINT)
             - --log-level=info
             - --nodeid=$(KUBE_NODE_NAME)
+            - --chroot-dir=/host
```

Comment:
Please template this like it is upstream. Talos does not support chroot, and allowing it to easily be disabled is convenient.